src/step-13-typescript/routes/login.ts

```diff
@@ -0,0 +1,34 @@
+import { Type, Static } from '@sinclair/typebox'
+import { FastifyInstance, FastifyRequest } from 'fastify'
+
+const BodySchema = Type.Object({
+  username: Type.String(),
+  password: Type.String(),
+})
+
+type BodySchema = Static<typeof BodySchema>
```

Comment:
this causes linting errors, but the linter is not running on typescript files. you need to change the lint npm script to provide --ext, otherwise it will only run on javascript files